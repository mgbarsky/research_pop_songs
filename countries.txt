Argentina
Australia
Austria
Belgium
Brazil
Bulgaria
Canada
Chile
China
Denmark
Finland
France
German
Greece
India
Ireland
Italy
Japan
Netherlands
New Zealand
Norway
Portugal
Russia
Spain
Sweden
Switzerland
UK Singles
Ukraine
USA Singles