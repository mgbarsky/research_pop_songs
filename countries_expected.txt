Japan
China
India
Russia
Ukraine
Bulgaria
Greece
Argentina
Chile
Brazil
Portugal
Spain
Italy
France
Belgium
Switzerland
Finland
Norway
Denmark
Sweden
Netherlands
Austria
German
Ireland
UK Singles
USA Singles
Canada
Australia
New Zealand
